crml/cennzx-spot/src/tests.rs

```diff
@@ -23,9 +23,201 @@ use crate::{
 	CoreAssetId, Error, Trait,
 };
 use core::convert::TryFrom;
-use frame_support::{traits::Currency, StorageValue};
-use mock::{AccountId, CennzXSpot, ExtBuilder, Origin, Test};
-use sp_core::{crypto::UncheckedInto, H256};
+use frame_support::{additional_traits::DummyDispatchVerifier, impl_outer_origin, traits::Currency, StorageValue};
```

Comment:
This shouldn't be here - It was meant to be pulled out and put into the mock. Hopefully a rebase fixes this?